includes/social-networks.php

```diff
@@ -166,4 +166,11 @@
 		'default' => "https://trello.com/b/{$board}",
 		'select'  => $board,
 	],
+	'unsplash'     => [
+                'icon'       => 'camera',
```

Comment:
@salvoventura Thanks for the PR! I think this line is using spaces - please use tab indentation.